CONTRIBUTING.md

```diff
@@ -130,41 +122,38 @@ Issue #190: Short summary of the issue
 ```
 
 Code
-====
+----
 
 This section include some tips that will help you to push source code.
 
-::: {.note}
-::: {.admonition-title}
-Note
-:::
-
-We recommend using [Python virtual
-environments](https://docs.python.org/3/tutorial/venv.html) to cleanly
-separate Qiskit from other applications and improve your experience.
-:::
-
-Setup with an environment
--------------------------
+**Note:**
+> We recommend using [Python virtual environments](https://docs.python.org/3/tutorial/venv.html) 
+> to cleanly separate Qiskit from other applications and improve your experience.
 
-The simplest way to use environments is by using Anaconda
+### Setup with Conda virtual environment
 
 ``` {.sh}
-conda create -y -n QiskitDevenv python=3
-source activate QiskitDevenv
+$ conda create -y -n QiskitDevenv python=3
+$ conda activate QiskitDevenv
+```
+
+### Setup with Python virtual environment
+```{.sh}
+$ python -m venv QiskitDevenv
+$ source QiskitDevenv/bin/activate (Linux and Mac)
```

Comment:
Can you separate out Linux/Mac and Windows commands instead of adding the `()` in the end? It's more lines but clearer and more consistent.